### PR TITLE
Unskipping remaining skipped tests in CSharp.UnitTests

### PR DESCRIPTION
**Pull request by VSadov**

### CODE REVIEW

**Comment by VSadov** [comment]
@dotnet/roslyn-compiler - please review

File: src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs [hunk 1]
```diff
@@ -2149,7 +2149,8 @@ private void CheckInternalMembers(NamedTypeSymbol type, bool isFromSource)
                 Assert.Null(member);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/6190"), WorkItem(90, "https://github.com/dotnet/roslyn/issues/90")]
+        [WorkItem(90, "https://github.com/dotnet/roslyn/issues/90")]
+        [Fact]
         public void EmitWithNoResourcesAllPlatforms()
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs]: It looks like sometimes this test is failing, I was hitting that failure too, but, I think, I was able to modify it to make it robust. See changes in #9870.

File: src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs [hunk 2]
```diff
@@ -3125,7 +3125,7 @@ public class C2 { }
 
         #region ComImportAttribute, CoClassAttribute
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/6190")]
+        [Fact]
         public void TestComImportAttribute()
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs]: It looks like this test is failing.

File: src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs [hunk 3]
```diff
@@ -2952,7 +2952,7 @@ static int F(string s)
         /// Unique ids should not conflict with ids
         /// from previous generation.
         /// </summary>
-        [Fact(Skip = "TODO")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/9847")]
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs]: Consider adding WorkItem attribute as well.

File: src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs [hunk 4]
```diff
@@ -5334,7 +5334,7 @@ .maxstack  2
         /// Should not re-use locals if the method metadata
         /// signature is unsupported.
         /// </summary>
-        [Fact(Skip = "TODO")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/9849")]
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs]: Consider adding WorkItem attribute as well.

File: src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs [hunk 5]
```diff
@@ -5418,7 +5418,7 @@ .maxstack  1
         /// <summary>
         /// Should not re-use locals with custom modifiers.
         /// </summary>
-        [Fact(Skip = "TODO")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/9848")]
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs]: Consider adding WorkItem attribute as well.

File: src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs (hunk 1 above)
Comment (VSadov) [inline src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs]: I see. It never failed for me so I did not know what could go wrong.
I hope the fix is sufficient.

File: src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs (hunk 3 above)
Comment (VSadov) [inline src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs]: done

File: src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs (hunk 4 above)
Comment (VSadov) [inline src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs]: done

File: src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs (hunk 5 above)
Comment (VSadov) [inline src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs]: done

**Comment by VSadov** [comment]
test prtest/win/dbg/unit32 please

**Comment by VSadov** [comment]
test prtest/mac/dbg/unit32 please
curl: (56) SSLRead() return error -9806

File: src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs [hunk 6]
```diff
@@ -2168,7 +2169,7 @@ private static void VerifyEmitWithNoResources(CSharpCompilation comp, Platform p
 
             using (var outputStream = new MemoryStream())
             {
-                var success = comp.WithOptions(options).Emit(outputStream).Success;
+                var success = comp.WithAssemblyName("EmitWithNoResourcesAllPlatforms_" + platform.ToString()).WithOptions(options).Emit(outputStream).Success;
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs]: Changing that wasn't sufficient for VB tests, they kept failing randomly until I switched to CompileAndVerify helper.

**Pull request merged**